FibonacciHeapTest.java

```diff
@@ -435,4 +442,164 @@ void simpleCase(){
             }
         }
     }
+
```

Comment:
here on puprose ?

Reply:
It's not the answers, it's just the code I used for measuring